SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -194,55 +194,55 @@ QueryConstraint InterestFactory::CreateNetCullDistanceConstraintWithFrequency(US
 	return CheckoutRadiusConstraintRoot;
 }
 
-TArray<Worker_ComponentId> InterestFactory::CreateClientNonAuthInterestResultType(USpatialClassInfoManager* ClassInfoManager)
+ResultType InterestFactory::CreateClientNonAuthInterestResultType(USpatialClassInfoManager* ClassInfoManager)
 {
-	TArray<Worker_ComponentId> ResultType;
+	ResultType ClientNonAuthResultType;
```

Comment:
This is the most confusing change you've made

Reply:
`ResultType` is a nicer type than `TArray<Worker_ComponentId>`, but I didn't want it to be `ResultType ResultType` because that reads strangely, so changed to this

Reply:
Added the type because I added a method to set the result type for a query depending on the flag because that functionality was duplicated everywhere, but as I wrote it I realised I would have had to write `TArray` twice just for this change so might as well make it its own type as it has meaning